swagger2.0/oic.sec.ael.swagger.json

```diff
@@ -0,0 +1,263 @@
+			"in": "query",
+			"name": "categoryfilter",
+			"required": false,
+			"type": "integer",
+			"enum": [
+				1, 2, 4, 8, 16, 32, 64
+			],
+			"x-example": 7
+		}
+	},
+	"definitions": {
+		"Aee": {
+			"description": "Auditable Event Entry logged by a Device",
+			"type": "object",
+			"properties": {
+				"aeid": {
+					"description": "Identity of the logged event",
+					"type": "string",
+					"readOnly": true
+				},
+				"devicetype": {
+					"description": "Device type which generated this Security Event (e.g. oic.d.light)",
+					"type": "array",
+					"minItems": 1,
+					"items": {
+						"type": "string",
+						"maxLength": 64
+					},
+					"readOnly": true
+				},
+				"di": {
+					"$ref": "https://openconnectivityfoundation.github.io/core/schemas/oic.types-schema.json#/definitions/uuid"
+				},
+				"category" : {
+					"description": "category of this Auditable Event: 1 (Access Control), 2 (Onboarding), 4 (Device), 8 (Authentication), 16 (SVR Modification), 32 (Cloud), 64 (Communication)",
+					"type": "integer",
+					"enum": [
+						1, 2, 4, 8, 16, 32, 64
+					],
+					"readOnly": true
+				},
+				"priority": {
+					"definitions": "priority of this Auditable Event: 0 (CRIT), 1 (ERR), 2 (WARN), 3 (INFO), 4 (DEBUG)",
+					"type": "integer",
+					"enum": [
+						0, 1, 2, 3, 4
+					], 
+					"readOnly": true
+				},
+				"timestamp": {
+					"description": "Time when this Auditable Event occured",
+					"type": "string",
+					"format": "date-time", 
+					"readOnly": true
+				},
+				"message": {
+					"description": "Description for this Auditable Event", 
+					"type": "string", 
+					"readOnly": true
+				},
+				"auxiliaryinfo": {
+					"description": "Supplementary info for Auditable Event message. (e.g. URI of specific Resource in ACE2 for 'Access Denied' message)",
+					"type": "array",
+					"minItems": 0, 
+					"items": {
+						"type": "string"
+					},
+					"readOnly": true
+				}
+			},
+			"required": [
+				"aeid", "category", "priority", "timestamp"
+			]
+		},
+
+		"Ael": {
+			"description": "Resource for storing Auditable Events List",
+			"type": "object",
+			"properties": {
+				"rt": {
+					"description": "Resource Type",
+					"type": "array",
+					"minItems": 1,
+					"uniqueItems": true,
+					"items": {
+						"maxLength": 64,
+						"type": "string",
+						"enum": [ "oic.r.ael" ]
+					},
+					"readOnly": true
+				},
+				"n": {
+					"$ref": "https://openconnectivityfoundation.github.io/core/schemas/oic.common.properties.core-schema.json#/definitions/n"
+				},
+				"id": {
+					"$ref": "https://openconnectivityfoundation.github.io/core/schemas/oic.common.properties.core-schema.json#/definitions/id"
+				},
+				"if": {
+					"description": "The OCF Interface set supported by this Resource",
+					"type": "array",
+					"minItems": 2,
+					"uniqueItems": true,
+					"items": {
+						"type": "string",
+						"enum": [ "oic.if.baseline", "oic.if.r" ]
```

Comment:
change to "oic.if.rw"

Reply:
Fixed in https://github.com/openconnectivityfoundation/security/pull/120